cmd/afsPmf/afsPmf.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/popgen"
+	"log"
+)
+
+func afsPmf(vcfFile string, outFile string, unPolarized bool) {
+	g, err := popgen.VcfToAfs(vcfFile, !unPolarized) //VcfToAfs is written in terms of polarized, so this is inverted here.
+	exception.FatalOnErr(err)
+	out := fileio.EasyCreate(outFile)
+
+	numAlleles := g.Sites[0].N
+	frequencySlice := make([]int, numAlleles)
+
+	for _, site := range g.Sites {
+		frequencySlice[site.I]++
```

Comment:
Is it worth it to have a check here that site.N == g.Sites[0].N

Reply:
Will fix in next commit